.github/workflows/buildAll.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  workflow_call:
+
+jobs:
+  build-all:
+    strategy:
+      matrix:
+        node_version: [lts/-1]
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node_version }}
+          cache: npm
+      - run: npm ci
+      - run: npm --version
+      - run: node --version
+      - run: npm run compile
+      - run: npm run vscode:package
+      - name: Stage Artifacts
+        run: |
+          mkdir extensions
+          find packages -name "*.vsix" -type f -exec cp {} ./extensions \;
+      - name: Upload Extensions
```

Comment:
I'd debated if we actually need to create and upload the vsix files for every commit on a PR.  I thought about just verifying we could build the vsix file and not saving the artifacts.  We could then have a manual step for saving the vsixes as artifacts if we actually want to install them.  It would save us much time but it would save us a ton of space as I think there's a limit in GHA.  Figured I could swing back to this update since we already do this for all builds in circle.

Reply:
yeah I was mostly debating if we actually need it...or if it could be on demand